sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -287,28 +282,28 @@ Protocols are those supported by the cURL library.</Description>
     </item>
     <item>
         <EFDB_Name>baudRate</EFDB_Name>
-        <Description>baud rate as a number</Description>
+        <Description>Baud rate as a number</Description>
```

Comment:
I imagine there are only certain permitted values?

Reply:
Ok, so you should state this in the description, no?

Reply:
It is there